webpack.config.js

```diff
@@ -0,0 +1,34 @@
+const webpack = require('webpack');
+
+module.exports = {
+  entry: {
+    common: './js/src/common.js',
+    documentation: './js/src/documentation.js',
+    install: './js/src/install.js',
+    nightly: './js/src/nightly.js',
+    search: './js/src/search.js',
```

Comment:
If you want to use Webpack, I think this should be done using code splitting... With this setup, some code required by two of the files (eg. if both `install.js` and `nightly.js` use the same module) will be duplicated across both bundles. Code splitting can pull the common code into a separate module that both can use.